packages/vue-instantsearch/src/components/Pagination.vue

```diff
@@ -107,14 +107,14 @@
             name="next"
             :createURL="() => state.createURL(state.currentRefinement + 1)"
             :is-last-page="state.isLastPage"
-            :refine="() => refine(state.currentRefinement + 1)"
+            :refine="() => refine(state.currentRefinement + 1, $event)"
           >
             <template v-if="!state.isLastPage">
               <a
                 :class="suit('link')"
                 aria-label="Next"
                 :href="state.createURL(state.currentRefinement + 1)"
-                @click.prevent="refine(state.currentRefinement + 1)"
+                @click.prevent="refine(state.currentRefinement + 1, $event)"
```

Comment:
can we use .exact? https://vuejs.org/guide/essentials/event-handling.html#key-modifiers

Reply:
Yes, it's actually much better! I forgot about this Vue magic.